src/viur/core/prototypes/list.py

```diff
@@ -350,7 +350,7 @@ def getDefaultListParams(self):
     @skey(allow_empty=True)
     def clone(self, key: db.Key | str | int, **kwargs):
         """
-        Clone an existing entry, and render the entry, eventually with error notes on incorrect data.
+        CloneBehavior an existing entry, and render the entry, eventually with error notes on incorrect data.
```

Comment:
Looks like a replace issue...
```suggestion
        Clone an existing entry, and render the entry, eventually with error notes on incorrect data.
```